irohad/ametsuchi/impl/postgres_options.cpp

```diff
@@ -5,12 +5,12 @@
 
 #include "ametsuchi/impl/postgres_options.hpp"
 
+#include <boost/algorithm/string.hpp>
```

Comment:
Are you sure that this is clang-format-7 applied? The idea is to have third-party library includes after standard library includes

Reply:
```
(881-no-drop-state-for-first-run)> clang-format -i -style=file irohad/ametsuchi/impl/postgres_options.cpp
(881-no-drop-state-for-first-run)> git sta
На ветке 881-no-drop-state-for-first-run
нечего коммитить, нет изменений в рабочем каталоге
```

Reply:
Yes sure

Reply:
https://jenkins.soramitsu.co.jp/job/iroha/job/iroha-hyperledger/job/PR-925/8/artifact/clang-format-report.txt
It's not clear that the clang-format version is correct

Reply:
clang-format version 11.1.0

Reply:
looks like versions are different.